news/1888.bugfix

```diff
@@ -0,0 +1 @@
+Allow sending `sendPasswordReset` if self registration is enabled. @MrTango
```

Comment:
This is a bit more accurate (and in plone.restapi release notes we always try to name the endpoint to give some context):

```suggestion
In the `@users` add endpoint, ignore the `sendPasswordReset` parameter if self registration is enabled. @MrTango
```